keyboards/system76/launch_1/rules.mk

```diff
@@ -0,0 +1,36 @@
+# MCU name
+MCU = atmega32u4
+
+# Allow flashing with USBasp
+AVRDUDE_MCU = $(MCU)
+
+# CPU frequency divided by two since AVR is at 3.3 V
+F_CPU = 8000000
+
+# External oscillator is 16 MHz
+F_USB = 16000000
+
+# Bootloader selection
+BOOTLOADER = atmel-dfu
+
+# Build options
+BOOTMAGIC_ENABLE = yes      # Bootmagic Lite
+MOUSEKEY_ENABLE = no        # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and system control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+DYNAMIC_KEYMAP_ENABLE = yes # Reconfigurable keyboard without flashing firmware
+NKRO_ENABLE = yes           # USB N-key rollover
+RAW_ENABLE = yes            # Raw HID commands (used by Keyboard Configurator)
+RGBLIGHT_ENABLE = no        # RGB backlight (conflicts with RGB matrix)
+RGB_MATRIX_ENABLE = yes     # RGB matrix
+RGB_MATRIX_DRIVER = WS2812
+RGB_MATRIX_CUSTOM_KB = yes  # Custom keyboard effects
+LTO_ENABLE = yes            # Link-time optimization for smaller binary
```

Comment:
```suggestion
# Build options
#   change yes to no to disable
BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
MOUSEKEY_ENABLE = no        # Mouse keys
EXTRAKEY_ENABLE = yes       # Audio control and System control
CONSOLE_ENABLE = no         # Console for debug
COMMAND_ENABLE = no         # Commands for debug and configuration
NKRO_ENABLE = yes           # Enable N-Key Rollover
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
AUDIO_ENABLE = no           # Audio output
RGB_MATRIX_ENABLE = yes
RGB_MATRIX_DRIVER = WS2812
RGB_MATRIX_CUSTOM_KB = yes
RAW_ENABLE = yes
DYNAMIC_KEYMAP_ENABLE = yes
LTO_ENABLE = yes
```

Reply:
Added back unused hardware definitions and explicitly disabled them.